docker-compose.yml

```diff
@@ -1,36 +1,39 @@
-version: '3.7'
+version: "3.7"
 
 services:
   db:
-    build:
-      context: .
-      dockerfile: docker/db.Dockerfile
+    image: postgres:latest
     volumes:
       - ./volumes:/var/lib/postgresql/data/
-    expose:
-      - "5432"
+    ports:
+      - "5433:5432"
```

Comment:
Why do you need to expose it on a different port?

Reply:
In order not to have conflicts with the portail database while debugging
But the services inside the SSO Docker network still communicate with the database through the 5432 port